docs/picongpu-docs-env.yml

```diff
@@ -0,0 +1,21 @@
+# Create a separate conda environment which fulfills the package requirements
+# for building the documentation
+# E.g. run `conda env create --file picongpu-docs-env.yml`
+#
+
+name: picongpu-docs-env
+channels:
+    - conda-forge
+dependencies:
+    - doxygen
```

Comment:
this duplicates the dependencies in `docs/requirements.txt` and now you will always have to update *two* positions instead of one.
Why is a pip virtual environment not sufficient?

Reply:
In order to avoid duplicating the entries that are already in `docs/requirements.txt`, you can try the following here:
```yaml
# ...
dependencies:
    - doxygen
    - pip
    - pip:
          - "-r requirements.txt"

Reply:
Thanks for the suggestion! That's a valid point :)
I'm gonna do it like that, then.